integr8ly/integreatly-operator
# Changes 3scale system app to use service endpoint

# Description
As we are rate limiting external requests to the 3scale backend listener any other managed 3scale components need to communicate with the backend listener via services.

## Verification steps

1. Install the operator from this branch
2. When the installation completes, verify the system app deployment config and check if the `APICAST_BACKEND_ROOT_ENDPOINT` and `BACKEND_ROUTE` envvar are using the `service_endpoint`
